src/components/MedCentersPage/MedCentersList/MedCenter/CenterStaff/CenterStaff.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { CenterStaff, StaffItem, StaffWrapper } from './CenterStaffStyle';
+
+interface IStaf {
```

Comment:
```suggestion
interface IStaff {
```
pls use the extension to check the spelling